tfjs-backend-webgpu/src/backend_webgpu.ts

```diff
@@ -452,11 +466,24 @@ export class WebGPUBackend extends KernelBackend {
 
     const encoder = this.device.createCommandEncoder();
     const pass = encoder.beginComputePass();
+    if (shouldTimeProgram) {
+      pass.writeTimestamp(this.querySet, this.cc); // --ForQuery
+    }
     pass.setPipeline(pipeline);
     pass.setBindGroup(0, bg);
     pass.dispatch(
         program.dispatch[0], program.dispatch[1], program.dispatch[2]);
+    if (shouldTimeProgram) {
+      pass.writeTimestamp(this.querySet, this.cc+1); // --ForQuery
+    }
     pass.endPass();
+
+    if (shouldTimeProgram) {
+      encoder.resolveQuerySet(this.querySet, this.cc, 2, this.dstBuffer, 8*this.cc); // --ForQuery
```

Comment:
8*this.cc is the offset in destionation buffer, if resolve two timestamps pre time, the offset should be 16 * this.cc.

Reply:
Oh, thanks for the comment.
I used this.cc++ twice, so no problem with 8 * this.cc, but this may missed when this is only one ops in model, I updated in another pr. https://github.com/haoyunfeix/tfjs/pull/30